stable/ambassador/README.md

```diff
@@ -142,6 +141,14 @@ Timings values have been removed in favor of setting the env variables using `en
 | `timing.drain`    | `AMBASSADOR_DRAIN_TIME`    |
 | `timing.shutdown` | `AMBASSADOR_SHUTDOWN_TIME` |
 
+### Ambassador ID
```

Comment:
Should probably move this up under a new section under Upgrading 

```
# Upgrading

## To 2.0.0

### Ambassador ID
...

## Migrating from `datawire/ambassador` chart (chart version 0.40.0 or 0.50.0)
...
```

Reply:
Makes sense. Fixing now.